src/modules/UI/scenes/CryptoExchange/CryptoExchangeConfirmTransactionModalComponent.js

```diff
@@ -12,7 +12,7 @@ import s from '../../../../locales/strings.js'
 import {GuiWallet} from '../../../../types'
 import THEME from '../../../../theme/variables/airbitz'
 
-type Props = {
+type CryptoExchangeConfirmTransactionModalOwnProps = {
   style: any,
```

Comment:
Please avoid using any
At the bare minimum, styles should be typed `Object` or `{}`

Reply:
If you see `any` when adding types, please replace them with more precise types